info/v2/conversion.go

```diff
@@ -155,6 +155,9 @@ func ContainerStatsFromV1(containerName string, spec *v1.ContainerSpec, stats []
 		if len(val.PerfStats) > 0 {
 			stat.PerfStats = val.PerfStats
 		}
+		if len(val.Resctrl.MemoryBandwidthMonitoring) > 0 || len(val.Resctrl.CacheMonitoringTechnology) > 0 {
```

Comment:
As I remember, you need add values from `Resctrl` also to `DeprecatedStatsFromV1` function to have them on API v2.0.
Have you checked that newly added values are available on http://127.0.0.1:8080/api/v2.0/stats?

Reply:
You are correct :)